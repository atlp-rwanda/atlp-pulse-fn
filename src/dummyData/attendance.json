[
  {
    "id": "1",
    "sprint": "Sprint 1",
    "session": "Demo",
    "record": "1"
  },
  {
    "id": "2",
    "sprint": "Sprint 2",
    "session": "Standup",
    "record": "2"
  },
  {
    "id": "3",
    "sprint": "Sprint 3",
    "session": "Demo",
    "record": "1"
  },
  {
    "id": "4",
    "sprint": "Sprint 4",
    "session": "E-level up",
    "record": "1"
  },
  {
    "id": "5",
    "sprint": "Sprint 5",
    "session": "Stand up",
    "record": "2"
  }
]
