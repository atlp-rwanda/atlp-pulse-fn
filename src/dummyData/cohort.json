[
  {
    "id": "1",
    "name": "Cohort 5",
    "coordinator": "john doe",
    "phase": "Core Concept",
    "startDate": "1st Jan 2022",
    "closeDate": "11th Dec 2022"
  },
  {
    "id": "2",
    "name": "Cohort 6",
    "coordinator": "john doe",
    "phase": "Team Project",
    "startDate": "1st Jan 2022",
    "closeDate": "11th Dec 2022"
  },
  {
    "id": "3",
    "name": "Cohort 7",
    "coordinator": "john doe",
    "phase": "Core Concept",
    "startDate": "1st Jan 2022",
    "closeDate": "11th Dec 2022"
  },
  {
    "id": "4",
    "name": "Cohort 8",
    "coordinator": "john doe",
    "phase": "Core Concept",
    "startDate": "1st Jan 2022",
    "closeDate": "11th Dec 2022"
  },
  {
    "id": "5",
    "name": "Cohort 9",
    "coordinator": "john doe",
    "phase": "Team project",
    "startDate": "1st Jan 2022",
    "closeDate": "11th Dec 2022"
  }
]
