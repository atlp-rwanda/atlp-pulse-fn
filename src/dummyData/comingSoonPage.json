[
  {
    "id": 1,
    "title": "Pricing page",
    "name": "Coming Soon",
    "content": "Something new is coming",
    "image": "placide.png"
  }
]
