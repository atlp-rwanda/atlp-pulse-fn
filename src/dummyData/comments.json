[
  {
    "profile": "https://cdn.pixabay.com/photo/2018/04/18/11/21/people-3330157_1280.jpg",
    "name": "Kenneth Ngabo",
    "createdAt": "Tuesday, 20:04",
    "comment": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam",
    "replies": [
      {
        "profile": "https://cdn.pixabay.com/photo/2018/02/16/14/38/portrait-3157821_1280.jpg",
        "createdAt": "Friday, 30th"
      },
      {
        "profile": "https://cdn.pixabay.com/photo/2016/11/22/21/42/woman-1850703_1280.jpg",
        "createdAt": "Friday, 10th"
      }
    ]
  },
  {
    "profile": "https://cdn.pixabay.com/photo/2019/04/18/10/49/africa-4136601_1280.jpg",
    "name": "Kenneth Ngabo2",
    "createdAt": "Wednesday, 16:21",
    "comment": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam",
    "replies": [
      {
        "profile": "https://cdn.pixabay.com/photo/2018/02/16/14/38/portrait-3157821_1280.jpg",
        "createdAt": "Monday, 27th"
      },
      {
        "profile": "https://cdn.pixabay.com/photo/2016/11/22/21/42/woman-1850703_1280.jpg",
        "createdAt": "Friday, 10th"
      }
    ]
  },
  {
    "profile": "https://cdn.pixabay.com/photo/2019/11/11/04/33/child-4617142_1280.jpg",
    "name": "Kenneth Ngabo3",
    "createdAt": "Monday, 02:30",
    "comment": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam",
    "replies": [
      {
        "profile": "https://cdn.pixabay.com/photo/2018/02/16/14/38/portrait-3157821_1280.jpg",
        "createdAt": "Thursday, 2nd"
      },
      {
        "profile": "https://cdn.pixabay.com/photo/2016/11/22/21/42/woman-1850703_1280.jpg",
        "createdAt": "Friday, 10th"
      }
    ]
  }
]
