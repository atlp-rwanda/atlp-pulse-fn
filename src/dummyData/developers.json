[
  {
    "name": "Keza Alice",
    "sprint": "1",
    "quantity": "2",
    "quality": "0",
    "professionalSkills": "2"
  },
  {
    "name": "Shema alain",
    "sprint": "5",
    "quantity": "2",
    "quality": "0",
    "professionalSkills": "2"
  },
  {
    "name": "Muneza James",
    "sprint": "3",
    "quantity": "2",
    "quality": "0",
    "professionalSkills": "2"
  },
  {
    "name": "Keza Alice",
    "sprint": "1",
    "quantity": "2",
    "quality": "0",
    "professionalSkills": "2"
  },
  {
    "name": "Keza Alice",
    "sprint": "1",
    "quantity": "2",
    "quality": "0",
    "professionalSkills": "2"
  },
  {
    "name": "Keza Alice",
    "sprint": "1",
    "quantity": "2",
    "quality": "0",
    "professionalSkills": "2"
  },
  {
    "name": "Keza Alice",
    "sprint": "1",
    "quantity": "2",
    "quality": "0",
    "professionalSkills": "2"
  },
  {
    "name": "Keza Alice",
    "sprint": "1",
    "quantity": "2",
    "quality": "0",
    "professionalSkills": "2"
  },
  {
    "name": "Keza Alice",
    "sprint": "1",
    "quantity": "2",
    "quality": "0",
    "professionalSkills": "2"
  },
  {
    "name": "Keza Alice",
    "sprint": "1",
    "quantity": "2",
    "quality": "0",
    "professionalSkills": "2"
  }
]
