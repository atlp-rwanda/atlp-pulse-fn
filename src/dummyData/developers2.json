[
  {
    "name": "Kevin Shema",
    "email": "keshem5@gmail.com",
    "rating": "1",
    "cohort": "1",
    "program": "Core concept"
  },
  {
    "name": "Muneza John",
    "email": "John@gmail.com",
    "rating": "2",
    "cohort": "1",
    "program": "Apprenticeship"
  },
  {
    "name": "Gatare Olivier",
    "email": "gatareo@gmail.com",
    "rating": "0",
    "cohort": "2",
    "program": "Team project"
  },
  {
    "name": "Kevin Uwitonze",
    "email": "keuwit@gmail.com",
    "rating": "1",
    "cohort": "2",
    "program": "Apprenticeship"
  },
  {
    "name": "Nsabimana Blaise",
    "email": "blaise@gmail.com",
    "rating": "2",
    "cohort": "3",
    "program": "Core concept"
  },
  {
    "name": "Damascene Kaneza",
    "email": "damkan@gmail.com",
    "rating": "2",
    "cohort": "3",
    "program": "Team project"
  },
  {
    "name": "Umutoni Alice",
    "email": "ali@gmail.com",
    "rating": "0",
    "cohort": "4",
    "program": "Team project"
  },
  {
    "name": "Ange Ishimwe",
    "email": "ishange@gmail.com",
    "rating": "1",
    "cohort": "4",
    "program": "Core concept"
  },
  {
    "name": "Keza Belinda",
    "email": "kezabe5@gmail.com",
    "rating": "1",
    "cohort": "5",
    "program": "Apprenticeship"
  },
  {
    "name": "Bwiza Gisele",
    "email": "bwiza@gmail.com",
    "rating": "2",
    "cohort": "5",
    "program": "Team project"
  }
]
