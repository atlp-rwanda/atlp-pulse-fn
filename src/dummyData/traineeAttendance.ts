export const traineeAttendanceData = [
  {
    week: '1',
    coordinatorId: '1234',
    trainees: [
      {
        traineeId: '12345',
        traineeEmail: 'adriana.griffin@devpulse.com',
        status: [
          { days: 'Monday', value: '2' },
          { days: 'Tuesday', value: '1' },
          { days: 'Wednesday', value: '2' },
          { days: 'Thusday', value: '0' },
        ],
      },
      {
        traineeId: '123456',
        traineeEmail: 'montoya.mckay@devpulse.com',
        status: [
          { days: 'Monday', value: '2' },
          { days: 'Tuesday', value: '1' },
          { days: 'Wednesday', value: '2' },
          { days: 'Thusday', value: '0' },
        ],
      },
      {
        traineeId: '1234567',
        traineeEmail: 'avery.travis@devpulse.com',
        status: [
          { days: 'Monday', value: '2' },
          { days: 'Tuesday', value: '1' },
          { days: 'Wednesday', value: '2' },
          { days: 'Thusday', value: '0' },
        ],
      },
      {
        traineeId: '12345678',
        traineeEmail: 'holman.knowles@devpulse.com',
        status: [
          { days: 'Monday', value: '2' },
          { days: 'Tuesday', value: '1' },
          { days: 'Wednesday', value: '2' },
          { days: 'Thusday', value: '0' },
        ],
      },
    ],
  },
];
