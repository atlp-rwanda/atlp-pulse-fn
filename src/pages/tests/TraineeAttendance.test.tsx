/* eslint-disable */
import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import renderer, { act } from 'react-test-renderer';
import { fireEvent, render, screen, waitFor } from '@testing-library/react';
import { MockedProvider as ApolloProvider } from '@apollo/client/testing';
import TraineeAttendanceTracker from '../TraineeAttendance';
import { traineeAttendanceData } from '../../dummyData/traineeAttendance';

describe('CRUD Of Trainee Attendance', () => {
  const mockOptional = [
    { value: 'Monday' },
    { value: 'Tuesday' },
    { value: 'Wednesday' },
    { value: 'Thusday' },
    { value: 'Friday' },
  ];

  it('Renders the TraineeAttendance Page', () => {
    const elem = renderer
      .create(
        <MemoryRouter>
          <ApolloProvider>
            <TraineeAttendanceTracker />
          </ApolloProvider>
        </MemoryRouter>,
      )
      .toJSON();
    expect(elem).toMatchSnapshot();
  });

  it('Test component', () => {
    render(
      <MemoryRouter>
        <ApolloProvider>
          <TraineeAttendanceTracker />
        </ApolloProvider>
      </MemoryRouter>,
    );

    const submitAttend = screen.getByTestId('submitAttend');
    const removeModel = screen.getByTestId('removeModel');
    const getDay = screen.getByTestId('getDay');
    const getWeek = screen.getByTestId('getWeek');
    const testWeek = screen.getByTestId('testWeek');

    fireEvent.click(submitAttend);
    fireEvent.click(removeModel);

    expect(screen.getByTestId('removeModel')).toBeInTheDocument();
    expect(getDay).toBeInTheDocument();
    expect(getWeek).toBeInTheDocument();
    expect(testWeek).toBeInTheDocument();

    fireEvent.change(testWeek, {
      target: {
        value: '1',
      },
    });

    fireEvent.change(getDay, { target: { value: 'Monday' } });
    fireEvent.change(getDay, { target: { value: 'Tuesday' } });
    fireEvent.change(getDay, { target: { value: 'Wednesday' } });
    fireEvent.change(getDay, { target: { value: 'Thursday' } });
    fireEvent.change(getDay, { target: { value: 'Friday' } });
  });
});
