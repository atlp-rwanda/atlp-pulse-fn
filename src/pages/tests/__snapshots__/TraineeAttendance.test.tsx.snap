// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CRUD Of Trainee Attendance Renders the TraineeAttendance Page 1`] = `
Array [
  <div
    className=" w-screen bg-black bg-opacity-30 backdrop-blur-sm min-h-[100%] fixed top-0 left-0 z-20 flex items-center justify-center px-4 hidden"
  >
    <div
      className="bg-white dark:bg-dark-bg w-full sm:w-3/4 md:w-1/2  xl:w-4/12 rounded-lg p-4 pb-8 "
    >
      <div
        className="card-title w-full flex  flex-wrap justify-center items-center  "
      >
        <h3
          className="font-bold text-sm dark:text-white text-center w-11/12"
        >
          Take Attendance
        </h3>
        <hr
          className=" bg-primary border-b my-3 w-full"
        />
      </div>
      <div
        className="card-body  bg-white dark:bg-dark-bg"
      >
        <form
          className=" py-3 px-8"
          data-testid="handleAddEvent"
          onSubmit={[Function]}
        >
          <div
            className="input my-3 h-12  felx flex-col"
          >
            <div
              className="grouped-input flex items-center h-full w-full rounded-md"
            >
              <select
                className="border-primary dark:bg-dark-tertiary dark:text-white border border-primary rounded outline-none px-5 font-sans text-xs py-2 w-full"
                data-testid="testWeek"
                defaultValue=""
                name="weeks"
                onBlur={[Function]}
                onChange={[Function]}
              >
                <option
                  value=""
                >
                  Select week
                </option>
                <option
                  value="1"
                >
                  Week 1
                </option>
                <option
                  value="2"
                >
                  Week 2
                </option>
                <option
                  value="3"
                >
                  Week 3
                </option>
                <option
                  value="4"
                >
                  Week 4
                </option>
              </select>
            </div>
             
            <p
              className="text-red-500 text-xs mb-8"
            />
          </div>
          <div
            className="input my-3 h-12  flex flex-col"
          >
            <div
              className="grouped-input flex items-center h-full  w-full rounded-md"
            >
              <select
                className="border-primary dark:bg-dark-tertiary dark:text-white border border-primary rounded outline-none px-5 font-sans text-xs py-2 w-full"
                data-testid="getDay"
                defaultValue=""
                name="days"
                onBlur={[Function]}
                onChange={[Function]}
              >
                <option
                  value=""
                >
                  Select day
                </option>
                <option
                  value="Monday"
                >
                  Monday
                </option>
                <option
                  value="Tuesday"
                >
                  Tuesday
                </option>
                <option
                  value="Wednesday"
                >
                  Wednesday
                </option>
                <option
                  value="Thursday"
                >
                  Thursday
                </option>
                <option
                  value="Friday"
                >
                  Friday
                </option>
              </select>
            </div>
            <p
              className="text-red-500 text-xs mb-4"
            />
          </div>
          <div
            className=" dark:bg-dark-tertiary dark:text-white overflow-y-auto h-[50vh] overflow-scroll scrollbar-hidden"
          >
            <table
              className="min-w-full divide-y divide-gray-200 dark:text-white overflow-scroll"
            >
              <thead
                className="dark:bg-dark-tertiary dark:text-white"
              >
                <tr>
                  <th
                    className="px-6 py-3 bg-gray-50 text-left text-xs leading-4 font-medium text-gray-500 uppercase tracking-wider dark:bg-dark-tertiary dark:text-white"
                  >
                    Trainee
                  </th>
                  <th
                    className="px-6 py-3 bg-gray-50 text-left text-xs leading-4 font-medium text-gray-500 uppercase tracking-wider dark:bg-dark-tertiary dark:text-white"
                  >
                    Score
                  </th>
                </tr>
              </thead>
              <tbody />
            </table>
          </div>
          <div
            className="w-full flex justify-between mx-10px"
          >
            <button
              className="py-2 w-[40%] md:w-1/3 bg-primary rounded font-sans text-sm text-white"
              data-testid="removeModel"
              onClick={[Function]}
              style={
                Object {
                  "height": "40px",
                }
              }
              type="button"
            >
              cancel
            </button>
            <button
              className="text-white py-2 w-[40%] md:w-1/3 bg-primary rounded"
              disabled={false}
              style={
                Object {
                  "height": "40px",
                }
              }
              type="submit"
            >
              Save
            </button>
          </div>
        </form>
      </div>
    </div>
  </div>,
  <div
    className="bg-light-bg dark:bg-dark-frame-bg lg:px-8 overflow-y-scroll pb-6"
  >
    <div
      className=""
    >
      <div
        className="flex items-center justify-between py-4 rounded-md w-full"
      >
        <div
          className="flex py-2 mt-2 "
        >
          <h2>
            Attendance Tracker
          </h2>
        </div>
        <div
          className="flex justify-left"
        >
          <select
            className="flex text-black px-4 py-2 rounded-md font-medium cursor-pointer border border-primary dark:bg-dark-tertiary dark:text-white"
            data-testid="getWeek"
            onChange={[Function]}
          >
            <option>
              Select Week
            </option>
            <option
              value="0"
            >
              Week 1
            </option>
            <option
              value="1"
            >
              Week 2
            </option>
            <option
              value="2"
            >
              Week 3
            </option>
            <option
              value="3"
            >
              Week 4
            </option>
          </select>
          <h2
            aria-hidden={true}
            className="mx-3 bg-primary px-4 py-2 rounded-md text-white font-medium cursor-pointer"
            data-testid="submitAttend"
            onClick={[Function]}
          >
            Submit Attendance
          </h2>
        </div>
      </div>
      <div
        className="bg-light-bg dark:bg-dark-frame-bg"
      >
        <div
          className=""
        >
          <div
            className="bg-white dark:bg-dark-bg shadow-lg px-5 py-8 rounded-md w-full"
          >
            <div
              className=""
            >
              <div
                className="flex items-center justify-between"
              />
            </div>
            <div>
              <div
                className="py-2 overflow-x-auto"
              >
                <div
                  className="w-full lg:min-w-full shadow rounded-lg  "
                >
                  <table
                    className="min-w-full leading-normal"
                  >
                    <thead
                      className="rounded-lg"
                    >
                      <tr
                        className=""
                      >
                        <th
                          className="border border-gray-300 p-2"
                          rowSpan={2}
                        >
                          Trainee
                        </th>
                        <th
                          className="border border-gray-300 p-2"
                          colSpan={5}
                        >
                          1
                        </th>
                      </tr>
                      <tr
                        className=""
                      >
                        <th
                          className="border border-gray-300 "
                        >
                          Mon
                        </th>
                        <th
                          className="border border-gray-300 p-2"
                        >
                          Tue
                        </th>
                        <th
                          className="border border-gray-300 p-2"
                        >
                          Wed
                        </th>
                        <th
                          className="border border-gray-300 p-2"
                        >
                          Thu
                        </th>
                        <th
                          className="border border-gray-300 p-2"
                        >
                          Fri
                        </th>
                      </tr>
                    </thead>
                    <p>
                      loading
                    </p>
                  </table>
                </div>
              </div>
            </div>
            <div
              className="flex items-center justify-center gap-1 mb-0"
            >
              <button
                className="page flex text-white h-12 w-12 items-center justify-center border-solid cursor-pointer bg-transparent disabled"
                data-testid="prev"
                onClick={[Function]}
                type="button"
              >
                ←
              </button>
              <button
                className="page flex text-white h-12 w-12 items-center justify-center border-solid cursor-pointer bg-transparent disabled"
                data-testid="page1"
                onClick={[Function]}
                type="button"
              >
                1
              </button>
              <button
                className="page flex text-white h-12 w-12 items-center justify-center border-solid cursor-pointer bg-transparent false"
                data-testid="page3"
                onClick={[Function]}
                type="button"
              >
                0
              </button>
              <button
                className="page flex text-white h-12 w-12 items-center justify-center border-solid cursor-pointer bg-transparent false"
                data-testid="next"
                onClick={[Function]}
                type="button"
              >
                →
              </button>
            </div>
          </div>
          <div
            className="big0 flex flex-row mt-4 gap-2"
          >
            <div
              className="big flex flex-col w-[50%] flex-1 gap-2"
            >
              <div
                className="bg-white flex flex-1 flex-col justify-center dark:bg-dark-bg border border-primary shadow-lg p-5 rounded-md w-full"
              >
                <h2
                  className="font-bold "
                >
                  Overview cohort 1
                </h2>
                <div>
                  <div
                    className="flex flex-row "
                  >
                    <div
                      className="flex flex-row justify-center items-center  "
                    >
                      <p
                        className="text-right font-semibold"
                      >
                        trainees:
                      </p>
                    </div>
                    <div
                      className="flex flex-row justify-center items-center  "
                    >
                      <p
                        className="text-right font-semibold"
                      >
                        0
                      </p>
                    </div>
                  </div>
                  <div
                    className="flex flex-row  "
                  >
                    <div
                      className="flex flex-row justify-center items-center "
                    >
                      <p
                        className="text-right font-semibold"
                      >
                        weeks:
                      </p>
                    </div>
                    <div
                      className="flex flex-row justify-center items-center "
                    >
                      <p
                        className="text-right font-semibold"
                      >
                         
                        1
                      </p>
                    </div>
                  </div>
                  <div
                    className="flex flex-row "
                  >
                    <div
                      className="flex flex-row justify-center items-center "
                    >
                      <p
                        className="text-right font-semibold"
                      >
                        Phase:
                      </p>
                    </div>
                    <div
                      className="flex flex-row justify-center items-center "
                    >
                      <p
                        className="text-right font-semibold"
                      >
                        II
                      </p>
                    </div>
                  </div>
                </div>
              </div>
              <div
                className="bg-white flex flex-1 flex-col justify-center dark:bg-dark-bg border border-primary shadow-lg p-5 rounded-md w-full"
              >
                <h2
                  className="font-bold mb-4 "
                >
                  Attendance Scores
                </h2>
                <div>
                  <div
                    className="flex flex-row "
                  >
                    <div
                      className="flex flex-row justify-center items-center "
                    >
                      <p
                        className="text-right font-semibold"
                      >
                        0
                      </p>
                    </div>
                    <div
                      className="flex flex-row justify-center items-center px-2 "
                    >
                      <p
                        className="text-right font-semibold"
                      >
                        Didn't attend and didn't communicate
                      </p>
                    </div>
                  </div>
                  <div
                    className="flex flex-row "
                  >
                    <div
                      className="flex flex-row "
                    >
                      <p
                        className="text-right font-semibold"
                      >
                        1:
                      </p>
                    </div>
                    <div
                      className="flex flex-row px-2"
                    >
                      <p
                        className="text-right font-semibold"
                      >
                        Didn't attend but communicated
                      </p>
                    </div>
                  </div>
                  <div
                    className="flex flex-row  "
                  >
                    <div
                      className="flex flex-row justify-center items-center "
                    >
                      <p
                        className="text-right font-semibold"
                      >
                        2:
                      </p>
                    </div>
                    <div
                      className="flex flex-row justify-center items-center px-2"
                    >
                      <p
                        className="text-right font-semibold"
                      >
                        Attended
                      </p>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              className="big2 bg-white flex flex-col justify-start  dark:bg-dark-bg border  border-primary shadow-lg p-2 rounded-md w-[50%]"
            >
              <h2
                className="font-bold ml-2 mb-4 "
              >
                Attendance Score Parcentage
              </h2>
              <h2
                className="font-bold ml-2"
              >
                Score Percentages (Percentages of trainee attendance at each score)
              </h2>
              <div
                className=""
              >
                <div
                  className="flex flex-row    px-8"
                >
                  <p
                    className="text-right font-semibold "
                  >
                    0:
                  </p>
                  <p
                    className="text-right ml-4"
                  >
                     
                    0
                    %
                  </p>
                </div>
                <div
                  className="flex flex-row px-8 "
                >
                  <p
                    className="text-right font-semibold"
                  >
                    1:
                  </p>
                  <p
                    className="text-right ml-4"
                  >
                     
                    0
                    %
                  </p>
                </div>
                <div
                  className="flex flex-row px-8"
                >
                  <p
                    className="text-right font-semibold"
                  >
                    2:
                  </p>
                  <p
                    className="text-right ml-4"
                  >
                     
                    0
                    %
                  </p>
                </div>
              </div>
              <div
                className=" w-[100%] h-[30%] flex flex-row  mt-5 justify-around items-center"
              >
                <div
                  className=" w-[100px] flex flex-col justify-center items-center"
                >
                  <svg
                    className="CircularProgressbar "
                    data-test-id="CircularProgressbar"
                    style={Object {}}
                    viewBox="0 0 100 100"
                  >
                    <path
                      className="CircularProgressbar-trail"
                      d="
      M 50,50
      m 0,-46
      a 46,46 0 1 1 0,92
      a 46,46 0 1 1 0,-92
    "
                      fillOpacity={0}
                      strokeWidth={8}
                      style={
                        Object {
                          "strokeDasharray": "289.02652413026095px 289.02652413026095px",
                          "strokeDashoffset": "0px",
                        }
                      }
                    />
                    <path
                      className="CircularProgressbar-path"
                      d="
      M 50,50
      m 0,-46
      a 46,46 0 1 1 0,92
      a 46,46 0 1 1 0,-92
    "
                      fillOpacity={0}
                      strokeWidth={8}
                      style={
                        Object {
                          "stroke": "#8667f2",
                          "strokeDasharray": "289.02652413026095px 289.02652413026095px",
                          "strokeDashoffset": "289.02652413026095px",
                        }
                      }
                    />
                    <text
                      className="CircularProgressbar-text"
                      style={
                        Object {
                          "fill": "#8667f2",
                        }
                      }
                      x={50}
                      y={50}
                    >
                      0%
                    </text>
                  </svg>
                  <label
                    htmlFor="0"
                  >
                     score 0
                  </label>
                </div>
                <div
                  className=" w-[100px] flex flex-col justify-center items-center"
                >
                  <svg
                    className="CircularProgressbar "
                    data-test-id="CircularProgressbar"
                    style={Object {}}
                    viewBox="0 0 100 100"
                  >
                    <path
                      className="CircularProgressbar-trail"
                      d="
      M 50,50
      m 0,-46
      a 46,46 0 1 1 0,92
      a 46,46 0 1 1 0,-92
    "
                      fillOpacity={0}
                      strokeWidth={8}
                      style={
                        Object {
                          "strokeDasharray": "289.02652413026095px 289.02652413026095px",
                          "strokeDashoffset": "0px",
                        }
                      }
                    />
                    <path
                      className="CircularProgressbar-path"
                      d="
      M 50,50
      m 0,-46
      a 46,46 0 1 1 0,92
      a 46,46 0 1 1 0,-92
    "
                      fillOpacity={0}
                      strokeWidth={8}
                      style={
                        Object {
                          "stroke": "#8667f2",
                          "strokeDasharray": "289.02652413026095px 289.02652413026095px",
                          "strokeDashoffset": "289.02652413026095px",
                        }
                      }
                    />
                    <text
                      className="CircularProgressbar-text"
                      style={
                        Object {
                          "fill": "#8667f2",
                        }
                      }
                      x={50}
                      y={50}
                    >
                      0%
                    </text>
                  </svg>
                  <label
                    htmlFor="0"
                  >
                     Score 1
                  </label>
                </div>
                <div
                  className=" w-[100px] flex flex-col justify-center items-center"
                >
                  <svg
                    className="CircularProgressbar "
                    data-test-id="CircularProgressbar"
                    style={Object {}}
                    viewBox="0 0 100 100"
                  >
                    <path
                      className="CircularProgressbar-trail"
                      d="
      M 50,50
      m 0,-46
      a 46,46 0 1 1 0,92
      a 46,46 0 1 1 0,-92
    "
                      fillOpacity={0}
                      strokeWidth={8}
                      style={
                        Object {
                          "strokeDasharray": "289.02652413026095px 289.02652413026095px",
                          "strokeDashoffset": "0px",
                        }
                      }
                    />
                    <path
                      className="CircularProgressbar-path"
                      d="
      M 50,50
      m 0,-46
      a 46,46 0 1 1 0,92
      a 46,46 0 1 1 0,-92
    "
                      fillOpacity={0}
                      strokeWidth={8}
                      style={
                        Object {
                          "stroke": "#8667f2",
                          "strokeDasharray": "289.02652413026095px 289.02652413026095px",
                          "strokeDashoffset": "289.02652413026095px",
                        }
                      }
                    />
                    <text
                      className="CircularProgressbar-text"
                      style={
                        Object {
                          "fill": "#8667f2",
                        }
                      }
                      x={50}
                      y={50}
                    >
                      0%
                    </text>
                  </svg>
                  <label
                    htmlFor="0"
                  >
                     Score 2
                  </label>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
]
`;
